### PR TITLE
CSP-1494 Do not create account are you sure page

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Employer.PR.Web/Models/Requests/DeclineCreateAccountViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Employer.PR.Web.Models.Requests;
+
+public class DeclineCreateAccountViewModel
+{
+    public Guid RequestId { get; set; }
```
Comment (DevenKShah) [inline src/SFA.DAS.Employer.PR.Web/Models/Requests/DeclineCreateAccountViewModel.cs]: RequestId is in the url, do you need it in the ViewModel?

File: src/SFA.DAS.Employer.PR.Web/Views/DeclineCreateAccount/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <p class="govuk-body">
+                @Model.ProviderName should have confirmed these details with you before sending the request to create an account. You can change your name by going back and selecting Change your name.
+            </p>
+            <p class="govuk-body">
+                If you select <strong>Do not create account</strong>, your apprenticeship service account will not be set up and we’ll remove the details added by @Model.ProviderName.
+                If the details were not correct, they can correct them and send you a new request.
+            </p>
+            <p class="govuk-body">
+            <form method="GET">
+                <input asp-for="RequestId" id="requestId" type="hidden" />
```
Comment (DevenKShah) [inline src/SFA.DAS.Employer.PR.Web/Views/DeclineCreateAccount/Index.cshtml]: Not sure why you need RequestId to post back? It should be there in the url?

**Review by DevenKShah — DISMISSED** [review]

File: src/SFA.DAS.Employer.PR.Web/Models/Requests/DeclineCreateAccountViewModel.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Employer.PR.Web/Models/Requests/DeclineCreateAccountViewModel.cs]: removed now

File: src/SFA.DAS.Employer.PR.Web/Views/DeclineCreateAccount/Index.cshtml (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.Employer.PR.Web/Views/DeclineCreateAccount/Index.cshtml]: removed now

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**